packages/xrpl/test/integration/fundWallet.test.ts

```diff
@@ -83,40 +83,6 @@ describe('fundWallet', function () {
   //   )
   // })
 
-  it(
```

Comment:
is this removed with an intent to remove support for the hooks testnet faucet? If so, can we include other files under this purview and also make a mention of it in the `HISTORY.md` file?

I took a stab at it here: https://github.com/ckeshava/xrpl.js/tree/removeHooksSupport

Reply:
Yes, we decided to no longer support Hooks testnet faucet. Can you create a PR with this included in your branch?

Reply:
ok. https://github.com/XRPLF/xrpl.js/pull/2711